packages/matter.js/src/behavior/definitions/identify/IdentifyServer.ts

```diff
@@ -18,4 +18,52 @@ const Base = IdentifyBehavior.for(Identify.Complete);
  * class for the features your implementation supports.
  */
 export class IdentifyServer extends Base {
+    protected declare internal: IdentifyServer.Internal;
+
+    override initialize() {
+        this.internal.identifyTimer = Time.getPeriodicTimer(
+            "Identify time update",
+            1000,
+            this.callback(this.#identifyTick),
+        );
+
+        // So whenever the attribute OR the identify command was invoked we react to it.
+        this.events.identifyTime$Change.on(() => {
```

Comment:
This should probably use reactTo

Reply:
Ok ... for info and understanding: why? ;-)

Reply:
Ok I think. got the reason ... we just read state so should be ok, but yes reactTo is better.